lib/features/thread/data/repository/thread_repository_impl.dart

```diff
@@ -136,6 +146,27 @@ class ThreadRepositoryImpl extends ThreadRepository {
     yield newEmailResponse;
   }
 
+  Future<EmailsResponse> _getFirstPage(
```

Comment:
We really need the output of this function? If not, please change the method as `_fillCacheWithFullFirstPage()`

Reply:
Yes, this function is used in the `refreshChanges` function, and in the `refreshChanges` function need output for update ui.